gobblin-metrics/src/main/java/gobblin/metrics/kafka/KafkaAvroSchemaRegistry.java

```diff
@@ -43,7 +42,7 @@
  *
  * @author ziliu
  */
-public class KafkaAvroSchemaRegistry {
+public class KafkaAvroSchemaRegistry implements KafkaSchemaRegistry<String, Schema> {
```

Comment:
The presence of KafkaAvroSchemaRegistry and KafkaSchemaRegistry in gobblin-metrics is a bit strange, isn't it?